lib/types/user-types.ts

```diff
@@ -3,7 +3,7 @@ import { User } from "@prisma/client";
 export interface FormOwner {
   id: string;
   email: string;
-  name?: string;
+  name?: string | null;
```

Comment:
What's the case for null here?

Reply:
in the new invitations lib PR, I've modified the code so we don't need this anymore.